src/components/KPop/KPop.vue

```diff
@@ -26,6 +26,7 @@
       <Transition name="kongponents-fade-transition">
         <div
           v-show="isVisible"
+          v-bind="popoverAttributes"
```

Comment:
issue: this seems a bit confusing. If you were to apply actual `PopoverAttributes` here, meaning something like `placement`, `hideCaret`, etc. they would not have the same functionality as if you set these properties on another component and passed them to the popover.